modules/op-resolver-hub.adoc

```diff
@@ -0,0 +1,188 @@
+// This module is included in the following assembly:
+//
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+
+:_content-type: PROCEDURE
+
+[id="resolver-hub_{context}"]
+= Specifying a remote pipeline or task from a Tekton catalog
+Using the Hub resolver, you can specify a remote pipeline or task that is defined in a public Tekton catalog, either link:https://artifacthub.io/[Artifact Hub] or link:https://hub.tekton.dev/[Tekton Hub].
+
+[id="resolver-hub-config_{context}"]
+== Configuring the Hub resolver
+
+If you want to change the default settings of the hub resolver, use the `TektonConfig` custom resource to configure the hub resolver.
+
+To edit the `TektonConfig` custom resource, enter the following command:
+
+[source,bash]
+----
+oc edit TektonConfig config
+----
+
+In the `TektonConfig` custom resource, in the `.spec.pipeline.hub-resolver-config` spec, specify  configuration, as in the following example:
```

Comment:
```suggestion
In the `TektonConfig` custom resource, specify the configuration in the `.spec.pipeline.hub-resolver-config` specification, as shown in the following example:
```

Reply:
"spec" is an accepted term here because of "spec:" in YAML